### PR TITLE
Documentation Mini Sprint TODO

### PR DESCRIPTION
**Pull request by tristantarrant**
Using the Pull Request mechanism to make reviewing / commenting easier
Once approved this should be merged
When tasks are resolved, the corresponding PRs should remove the relevant item from this document
This is virtuall assigned to everybody.

### CODE REVIEW

File: documentation/TODO.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+### Default values for properties (Ryan)
+* Remove this chapter and find a way to generate the information into the configuration reference
+
+### HTTP session clustering and caching (Sebastian)
+* Move under an overarching Integrations chapter
+* Remove the  open tasks
+* Mention Spring Session
+* Eventually will also contain Paul’s Tomcat adapter
+
+### Hadoop + Spark
```
Comment (gustavonalle) [inline documentation/TODO.md]: How about link to https://github.com/infinispan/infinispan-spark/blob/master/README.md and  https://github.com/infinispan/infinispan-hadoop/blob/master/README.md ?

File: documentation/TODO.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+* Ensure that all parameters are documented with the @param doclet
+* Can this be somewhat automated via checkstyle  ?
+
+## FAQs
+* We get many recurring questions on the forum / stackoverflow / IRC / etc. Try to distill some of the most significant ones you have answered successfully.
+* Even questions that are related to product (sme-jdg, customer portal, etc) can be of use (obviously purged from any product reference)
+* Things we CANNOT do are also important, and we should point out any limitations
+
+## Performance Guide (Sebastian)
+* This should be linked in the appropriate places from the User Guide
```
Comment (slaskawi) [inline documentation/TODO.md]: This one can be removed. The most of the work is already done: https://github.com/infinispan/infinispan/pull/4540

Comment (gustavonalle) [inline documentation/TODO.md]: There's still a lot of ground to cover for query performance guide, I'll add it next week

**Comment by tristantarrant** [comment]
Pushed.

**Pull request merged**